ocamltest/Makefile

```diff
@@ -202,19 +202,24 @@ all: ocamltest$(EXE)
 allopt: ocamltest.opt$(EXE)
 opt.opt: allopt
 
-ocamltest$(EXE): $(bytecode_modules)
-	$(ocamlc_cmd) -custom ocamlcommon.cma ocamlbytecomp.cma -o $@ $^
+compdeps_byte=\
+  $(ROOTDIR)/compilerlibs/ocamlcommon.cma \
+  $(ROOTDIR)/compilerlibs/ocamlbytecomp.cma
+compdeps_opt=$(compdeps_byte:.cma=.cmxa)
```

Comment:
Possibly (I personally prefer the explicit string functions to the substitution syntax):

```make
compdeps=ocamlcommon ocamlbytecomp
compdeps_archives=$(addprefix $(ROOTDIR)/compilerlibs/,$(compdeps))
compdeps_byte=$(addsuffix .cma,$(compdeps_archives))
compdeps_opt=$(addsuffix .cmxa,$(compdeps_archives))
```

Reply:
The extension-replacement approach is commonly used in this makefile already, so I would rather keep it for consistenty. I will use the addprefix trick. (Personally I'm not sure the conceptual complexity is worth it, but using it is also very much in line with Sébastien's style, so it's a good suggestion.)

Reply:
In the end I took the whole suggestion -- with a longer variable name for `compdeps` to avoid making it too easy to use from a rule by mistake.